tools/readme.md

```diff
@@ -79,4 +79,16 @@ Script that reduces code for a hang/false positive.
 
 Script to generate a `times.log` file that contains timing information of the last 20 revisions.
 
+### * tools/donate-cpu.py
 
+Script to donate a CPU time to Cppcheck project.
```

Comment:
`Script to donate CPU time to Cppcheck project by checking current Debian packages.` ?

Reply:
The main idea of this line is a donation of CPU time. I could add information about Debian packages if you insists. But it's a part of implementation and could be changed in the future.

Reply:
I don't care about the Linux distribution. But it should document what the CPU is used for. Running regression tests on the software or digging bitcoins for the developers or ...

Reply:
I thought keep it brief but it's a good point. Thank you.